barretenberg/cpp/pil/vm2/keccakf1600.pil

```diff
@@ -1,8 +1,424 @@
+include "bitwise.pil";
+include "constants_gen.pil";
+
 namespace keccakf1600;
 
+// TODO: Documentation and explanations of some tricks like for the bit circular rotation which requires
+//       a single range check.
+
+// TODO: Remove the dummy values below
+
 // Dummy.
 pol commit keccak[1000];
 
 // Dummy.
 keccak[0] = keccak[0];
 keccak[1] = keccak[1];
+
+
+// keccakf1600 selector
+pol commit sel;
+sel * (1 - sel) = 0;
+
+// No relations will be checked if this identity is satisfied.
+#[skippable_if]
+sel = 0;
+
+// TODO: We need this temporarily while we do not allow for aliases in the lookup tuple
+pol commit bitwise_xor_op_id;
+sel * (bitwise_xor_op_id - constants.AVM_BITWISE_XOR_OP_ID) = 0;
+
+// keccakf1600 permutation state corresponds to 25 64-bit words. They are indexed by two
+// indices ranging over 0,1,2,3,4.
+// Each state word evolves over the different transformations and a state word is prefixed by
+// "state" and suffixed bye the corresponding indices.
+
+// State words before starting the round.
+pol commit state_in_00, state_in_01, state_in_02, state_in_03, state_in_04,
```

Comment:
DId you try using the array pattern for these types?
`pol commit state_0[4];` etc

This might also apply to the results of the xors - `theta_xor[...]`;